src/app/pages/services/services-table.component.html

```diff
@@ -39,15 +39,15 @@
       <span class="datatable-header-cell-label">Start Automatically</span>
     </ng-template>
     <ng-template let-row="row" let-value="value" ngx-datatable-cell-template>
-      <mat-checkbox id="checkbox__{{row.label}}" class="checkbox" style="position:absolute;bottom:16px;" [(ngModel)]="row.enable" (change)="this.conf.enableToggle($event, row)"></mat-checkbox>
+      <mat-checkbox id="{{row.label}}_StartAutomatically_checkbox" class="checkbox" style="position:absolute;bottom:16px;" [(ngModel)]="row.enable" (change)="this.conf.enableToggle($event, row)"></mat-checkbox>
```

Comment:
This should have a space to match the exact text of the column title:
```suggestion
      <mat-checkbox id="{{row.label}}_Start Automatically_checkbox" class="checkbox" style="position:absolute;bottom:16px;" [(ngModel)]="row.enable" (change)="this.conf.enableToggle($event, row)"></mat-checkbox>
```